GUI/Controls/ModInfoTabs/Metadata.cs

```diff
@@ -72,6 +72,19 @@ public void UpdateModInfo(GUIMod gui_module)
                     ReplacementTextBox.Text = module.replaced_by.ToString();
                 }
 
+                if (gui_module.DownloadCount == null || gui_module.DownloadCount == 0)
+                {
+                    DownloadCountLabel.Visible = false;
+                    DownloadCountTextBox.Visible = false;
+                    MetadataTable.LayoutSettings.RowStyles[3].Height = 0;
+                }
+                else
+                {
+                    DownloadCountLabel.Visible = true;
+                    DownloadCountTextBox.Visible = true;
+                    MetadataTable.LayoutSettings.RowStyles[3].Height = 0;
```

Comment:
The height of the row should be 30 when the count is supposed to be visible:

```suggestion
                    MetadataTable.LayoutSettings.RowStyles[3].Height = 30;
```